backend/app/main.py

```diff
@@ -0,0 +1,41 @@
+from fastapi import FastAPI
+from contextlib import asynccontextmanager
+
+from app.routers.records import router 
+from app.storage.storage import storage as wikipedia_storage
+
+import os
+
+@asynccontextmanager 
+async def lifespan(app: FastAPI):
+
+    """
+    A lifespan event to read in the Wikipedia output data and persist it in storage.
+    """
+        
+    file_path = os.getcwd()+"\\app\\data\\mini-wikipedia.output.txt"
+    
+    wikipedia_storage.read_wikipedia_data(file_path)
+    wikipedia_storage.set_current_title(first_title = True)
+    
+    app.state.wikipedia_storage = wikipedia_storage
+
+    yield 
+
+    app.state.wikipedia_storage.reset()
+
+app = FastAPI(lifespan = lifespan)
+app.include_router(router)
+
+@app.get("/")
+async def root():
+
+    """
+    A path operation function of the root endpoint.
+    It returns the first Wikipedia record stored in wikipedia_storage.  
+    """
+
+    return app.state.wikipedia_storage.get_current_record()
```

Comment:
API should have proper URL structure, so the result type is guessable from the structure e.g., `/api/v1/wikipedia/current` or similar.